src/tokenize.js

```diff
@@ -548,13 +548,38 @@ pp.readString = function(quote) {
 
 // Reads template string tokens.
 
+const INVALID_TEMPLATE_ESCAPE_ERROR = {}
+
+pp.tryReadTemplateToken = function() {
+  this.inTemplateElement = true
+  try {
+    this.readTmplToken()
+  } catch (err) {
+    if (err === INVALID_TEMPLATE_ESCAPE_ERROR) {
+      this.readInvalidTemplateToken()
+    } else {
+      throw err
+    }
+  }
+
+  this.inTemplateElement = false
```

Comment:
It's true that this will be skipped over, but does this matter since a parsing error gets thrown anyway? My understanding was that the `Parser` instance is discarded when a syntax error occurs.